docs/Manual_Install.md

```diff
@@ -180,13 +180,6 @@ Log onto any node in the deployment and create the file `/etc/clearwater/shared_
     ellis_api_key=<secret>
     ellis_cookie_key=<secret>
 
-If you wish to enable the optional I-CSCF function, also add the following:
```

Comment:
Should we document something about disabling I-CSCF function? I'm not too fussed.